tests/lib/rules/new-cap.js

```diff
@@ -59,7 +59,8 @@ ruleTester.run("new-cap", rule, {
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Bar"] }] },
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Foo.Bar"] }] },
         { code: "var x = new foo.bar(42);", options: [{ newIsCapExceptions: ["bar"] }] },
-        { code: "var x = new foo.bar(42);", options: [{ newIsCapExceptions: ["foo.bar"] }] }
+        { code: "var x = new foo.bar(42);", options: [{ newIsCapExceptions: ["foo.bar"] }] },
```

Comment:
Can you add a test with `properties` and `capIsNew`? `isCapAllowed` that you modified is called for both, and I want to make sure we have tests for that.